src/test/java/seedu/planner/logic/parser/module/ModuleAddCommandParserTest.java

```diff
@@ -16,7 +19,9 @@
 
     @Test
     public void parse_validArgs_returnsAddCommand() {
-        assertParseSuccess(parser, "CS2040", new ModuleAddCommand(new ModuleCode("CS2040")));
+        List<ModuleCode> moduleCodes = new ArrayList<>();
+        moduleCodes.add(new ModuleCode("CS2040"));
+        assertParseSuccess(parser, "CS2040", new ModuleAddCommand(moduleCodes));
```

Comment:
Could let `ModuleAddCommand` accept a single `ModuleCode` as well. This will reduce the number of test cases that have to be changed.

Reply:
It already does, will revert the change in the test to reflect the original

Reply:
I had to swap your class attribute ```ModuleCode moduleCode``` to ```List<ModuleCode> moduleCodes```

as such, I have updated the default constructor to store the single ModuleCode in the List as shown:
```
public ModuleAddCommand(ModuleCode moduleCode) {
        requireAllNonNull(moduleCode);
        this.moduleCodes = Arrays.asList(moduleCode);
    }
```